app/models/profile.rb

```diff
@@ -21,10 +21,12 @@ class Profile < ApplicationRecord
   # rubocop:disable Rails/SkipsModelValidations
   def activate
     now = Time.zone.now
+    is_reproof = Profile.find_by(user_id: user_id, active: true)
```

Comment:
I'd love to re-use `user.active_profile` (except that does so memoization :/)

so then we could do:

```
had_active_profile = user.active_profile.present?
```